src/OrleansProviders/Streams/Memory/MemoryAdapterConfig.cs

```diff
@@ -10,17 +10,37 @@ namespace Orleans.Providers.Streams.Memory
     /// </summary>
     public class MemoryAdapterConfig
     {
+        /// <summary>
+        /// Stream provider name.
+        /// </summary>
         public String StreamProviderName { get; private set; }
 
+        /// <summary>
+        /// Total queue count name. Indicates the name of the property in provider config.
+        /// </summary>
         private const string TotalQueueCountName = "TotalQueueCount";
```

Comment:
Having it public makes it easier for developers that want to specify the value in xml configs to do so.  I'm not saying it's entirely necessary, just that when we've hidden config value names in the past it's made it difficult for end users to configure systems correctly.